packages/hs-react-ui/src/components/InteractionFeedback/InteractionFeedback.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useCallback, useState } from 'react';
+import { animated, useTransition } from 'react-spring';
+import styled, { StyledComponentBase } from 'styled-components';
+import shortid from 'shortid';
+import colors from '../../enums/colors';
+import useResizeObserver from 'use-resize-observer/polyfilled';
+
+// TODO Reduce amount of any/ts-ignore done here
+
+const Circle = styled.svg`
```

Comment:
Propose naming `Circle` to `SVGContainer` or something similar, expose it as a subcomponent, and then rename `fragment` to `shape`
```suggestion
const SvgContainer = styled.svg`
```